borsar/viz.py

```diff
@@ -347,7 +347,7 @@ def _extract_topo_channels(ax):
         # if there are no circles: look for PathCollection
         path_collection = ax.findobj(mpl.collections.PathCollection)
         if len(path_collection) > 0:
-            chans = path_collection[0]
+            chans = path_collection[8]
```

Comment:
not sure why this was needed to make the example build :shrug:

Reply:
thanks, I'll take a look at this after merging. What mne version did you use?

Reply:
I used 1.0.2, installed in a fresh conda environment via pip